src/main/kotlin/ink/abb/pogo/scraper/tasks/HatchEggs.kt

```diff
@@ -38,7 +38,9 @@ class HatchEggs : Task {
         val incubators = ctx.api.cachedInventories.incubators
         val eggs = ctx.api.cachedInventories.hatchery.eggs
 
-        val freeIncubators = incubators.filter { !it.isInUse }
+        val freeIncubators = incubators
+                .filter { !it.isInUse }
+                .sortedByDescending { it.usesRemaining }
```

Comment:
That is what I also discovered, therefore the sort works perfectly without any additional code! (: